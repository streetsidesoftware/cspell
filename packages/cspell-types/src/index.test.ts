import * as index from './index';

describe('Validate that the types build', () => {
    test('MessageTypes should be exported.', () => {
        expect(index).toHaveProperty('MessageTypes');
    });
    test('Make sure exports do not change.', () => {
        expect(Object.keys(index).sort()).toMatchSnapshot();
    });
});
