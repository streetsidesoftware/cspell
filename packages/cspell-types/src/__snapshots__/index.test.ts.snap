// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Validate that the types build Make sure exports do not change. 1`] = `
Array [
  "ConfigFields",
  "MessageTypes",
]
`;
